compose/runtime/runtime/build.gradle

```diff
@@ -115,7 +115,7 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
         sourceSets {
             commonMain.dependencies {
                 implementation(libs.kotlinStdlib)
-                implementation(libs.kotlinCoroutinesCore)
+                api(libs.kotlinCoroutinesCore)
```

Comment:
Should it be changed in AOSP too?

Reply:
Yes, when they start publishing compose runtime for all kotlin targets.

I remember you're working on adoptin the build.gradle scripts from aosp, right? 
Will it be a potential place with merge conflict then?

Reply:
Got it. I'll create a CL

Reply:
It's already api() in AOSP. https://android-review.googlesource.com/c/platform/frameworks/support/+/3443339

Probably we didn't get this change yet.